src/ScrumBoardItBundle/Services/ApiGitHub.php

```diff
@@ -0,0 +1,258 @@
+<?php
+namespace ScrumBoardItBundle\Services;
+
+use ScrumBoardItBundle\Services\AbstractApi;
+use Symfony\Component\HttpFoundation\Request;
+use ScrumBoardItBundle\Entity\Issue\Task;
+use ScrumBoardItBundle\Entity\Issue\SubTask;
+use ScrumBoardItBundle\Entity\Issue\IssueInterface;
+use ScrumBoardItBundle\Form\Type\Search\GithubSearchType;
+
+/**
+ *
+ * @author Brieuc Pouliquen <brieuc.pouliquen@canaltp.fr>
+ *        
+ */
+class ApiGitHub extends AbstractApi
+{
+
+    /**
+     *
+     * {@inheritdoc}
+     *
+     */
+    public function getProjects()
+    {
+        $api = $this->getProjectApi();
+        $data = $this->call($api);
+        $projects = array();
+        foreach ($data as $project) {
+            $projects['Propriétaire: ' . $project->owner->login][$project->name] = $project->full_name;
```

Comment:
Remove the "Propriétaire: " string in the index, it's useless and has to be managed by the form